examples/cloudflare/README.md

```diff
@@ -1,41 +1,56 @@
 # F1 Stats on Cloudflare Workers
+
 This is an example Cloudflare Worker that connects to a PlanetScale database to update and retrieve Formula 1 stats.
 It has two parts: a `cron` that runs to update the database with stats. And a `data.json` endpoint that serves the stats up in JSON format.
 
 ## Setup
+
 1. Run `npm install`.
 2. Create a PlanetScale database to store the stats. Add the schema shown below.
-3. Enroll your new database in the **PlanetScale serverless driver for JavaScript** beta (under your database's **Settings** > **Beta features** section).
 3. Create a PlanetScale password and add the new credentials as secrets:
-- `npx wrangler secret put PSCALE_HOST`
-- `npx wrangler secret put PSCALE_PASSWORD`
-- `npx wrangler secret put PSCALE_USERNAME`
+
+- `npx wrangler secret put DATABASE_HOST`
+- `npx wrangler secret put DATABASE_PASSWORD`
+- `npx wrangler secret put DATABASE_USERNAME`
+- `npx wrangler secret put REPLICA_DATABASE_HOST`
+- `npx wrangler secret put REPLICA_DATABASE_PASSWORD`
+- `npx wrangler secret put REPLICA_DATABASE_USERNAME`
+
 4. Run `npx wrangler dev` to start the app.
 
 ### Running the cron
+
 1. To connect to the database when running the app locally you'll need to add the connection details to your environment. One way to do this is to add them to `wrangler.toml`, just be sure not to check them in to your Git repo.
-  ```
-  [vars]
-  PSCALE_USERNAME = "USERNAME"
-  PSCALE_PASSWORD = "PASSWORD"
-  PSCALE_HOST = “REGION.connect.psdb.cloud”
-  ```
+
+```
+[vars]
+DATABASE_USERNAME = "USERNAME"
+DATABASE_PASSWORD = "PASSWORD"
+DATABASE_HOST = “REGION.connect.psdb.cloud”
+REPLICA_DATABASE_USERNAME = "USERNAME"
+REPLICA_DATABASE_PASSWORD = "PASSWORD"
+REPLICA_DATABASE_HOST = “REGION.connect.psdb.cloud”
```

Comment:
Nit: This is a bit more important, but it should be provider (`aws` or `gcp`, I believe) for this env var.

```suggestion
REPLICA_DATABASE_HOST = “PROVIDER.connect.psdb.cloud”
```